www/app/main.ts

```diff
@@ -45,6 +45,18 @@ function handleInputs(map: L.Map) {
     maxIterations = parsedValue;
     refreshMap(map);
   }, 1000);
+  
+  const exponentInput = <HTMLInputElement>document.getElementById("exponent");
+  exponentInput.value = String(exponent);
+  exponentInput.oninput = debounce(({ target }) => {
```

Comment:
Do you think the map zoom/position should be reset on exponent change?

Reply:
Maybe so!! Although I feel like it should probably have the same behavior as tuning iterations? It can be pretty interesting anyway to explore what the same region looks like with different exponents

Reply:
Okay fair enough, we'll keep it the way it is. Looks like this is ready to merge!